diff --git a/debian/NEWS b/debian/NEWS
index b305d35..ed047d7 100644
--- a/debian/NEWS
+++ b/debian/NEWS
@@ -1,22 +1,22 @@
-exim4 (4.94-99a) jessie; urgency=medium
+exim4 (4.94-99a) wheezy; urgency=medium
 
   * https://github.com/Exim/exim/blob/exim-4.94%2Bfixes/doc/doc-txt/ChangeLog
 
  -- kaut <maksimkaut92@gmail.com>  Tue, 11 Aug 2020 18:00:00 +0200
 
-exim4 (4.92.3-99a) jessie; urgency=medium
+exim4 (4.92.3-99a) wheezy; urgency=medium
 
   * https://github.com/Exim/exim/blob/exim-4.92.3%2Bfixes/doc/doc-txt/ChangeLog
 
  -- s3rj1k <evasive.gyron@gmail.com>  Mon, 30 Sep 2019 09:00:00 +0200
 
-exim4 (4.92.2-99a) jessie; urgency=medium
+exim4 (4.92.2-99a) wheezy; urgency=medium
 
   * https://github.com/Exim/exim/blob/exim-4.92.2%2Bfixes/doc/doc-txt/ChangeLog
 
  -- kaut <maksimkaut92@gmail.com>  Mon, 9 Aug 2019 12:00:00 +0100
 
-exim4 (4.92.1-99a) jessie; urgency=medium
+exim4 (4.92.1-99a) wheezy; urgency=medium
 
   * https://github.com/Exim/exim/blob/exim-4.92.1%2Bfixes/doc/doc-txt/ChangeLog
 
diff --git a/debian/changelog b/debian/changelog
index 5d6211d..aa8aa2f 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,28 +1,28 @@
-exim4 (4.94-99b) jessie; urgency=medium
+exim4 (4.94-99b) wheezy; urgency=medium
 
   * Rebuild
 
  -- kaut <maksimkaut92@gmail.com>  Tue, 27 Aug 2020 12:00:00 +0200
 
-exim4 (4.94-99a) jessie; urgency=medium
+exim4 (4.94-99a) wheezy; urgency=medium
 
   * https://github.com/Exim/exim/blob/exim-4.94%2Bfixes/doc/doc-txt/ChangeLog
 
  -- kaut <maksimkaut92@gmail.com>  Tue, 11 Aug 2020 18:00:00 +0200
 
-exim4 (4.92.3-99a) jessie; urgency=medium
+exim4 (4.92.3-99a) wheezy; urgency=medium
 
   * https://github.com/Exim/exim/blob/exim-4.92.3%2Bfixes/doc/doc-txt/ChangeLog
 
  -- s3rj1k <evasive.gyron@gmail.com>  Mon, 30 Sep 2019 09:00:00 +0200
 
-exim4 (4.92.2-99a) jessie; urgency=medium
+exim4 (4.92.2-99a) wheezy; urgency=medium
 
   * https://github.com/Exim/exim/blob/exim-4.92.2%2Bfixes/doc/doc-txt/ChangeLog
 
  -- kaut <maksimkaut92@gmail.com>  Mon, 9 Aug 2019 12:00:00 +0100
 
-exim4 (4.92.1-99a) jessie; urgency=medium
+exim4 (4.92.1-99a) wheezy; urgency=medium
 
   * https://github.com/Exim/exim/blob/exim-4.92.1%2Bfixes/doc/doc-txt/ChangeLog
 
diff --git a/debian/compat b/debian/compat
index ec63514..7f8f011 100644
--- a/debian/compat
+++ b/debian/compat
@@ -1 +1 @@
-9
+7
diff --git a/debian/control b/debian/control
index 2dd57f9..3998d5f 100644
--- a/debian/control
+++ b/debian/control
@@ -3,13 +3,13 @@ Section: mail
 Priority: standard
 Maintainer: s3rj1k <evasive.gyron@gmail.com>
 Homepage: https://www.exim.org/
-Standards-Version: 3.9.8
+Standards-Version: 3.9.3
 Build-Depends:
- debhelper (>= 9),
+ debhelper (>= 7),
  default-libmysqlclient-dev | libmysqlclient-dev | libmysqlclient15-dev | libmysqlclient18-dev | libmysqlclient20-dev,
  docbook-xml,
  docbook-xsl,
- libdb5.3-dev,
+ libdb5.1-dev,
  libgnutls28-dev (>= 3.5.7) | libssl-dev,
  libident-dev,
  libpam0g-dev,
diff --git a/debian/rules b/debian/rules
index b314438..3287e45 100755
--- a/debian/rules
+++ b/debian/rules
@@ -52,7 +52,7 @@ PROVIDE_DEFAULT_MTA := $(shell if dpkg-vendor --is Ubuntu || \
 	echo "default-mta" ; fi)
 # for reproducible build. If set exim would use $TZ as default value for
 # TIMEZONE_DEFAULT
-undefine TZ
+#undefine TZ
 unexport TZ
 
 
