diff --git a/debian/NEWS b/debian/NEWS
index 91a6dc3..d65f363 100644
--- a/debian/NEWS
+++ b/debian/NEWS
@@ -1,10 +1,10 @@
-exim4 (4.92.2-99a) jessie; urgency=medium
+exim4 (4.92.2-99a) stretch; urgency=medium
 
   * https://github.com/Exim/exim/blob/exim-4.92.2%2Bfixes/doc/doc-txt/ChangeLog
 
  -- kaut <maksimkaut92@gmail.com>  Mon, 9 Aug 2019 12:00:00 +0100
 
-exim4 (4.92.1-99a) jessie; urgency=medium
+exim4 (4.92.1-99a) stretch; urgency=medium
 
   * https://github.com/Exim/exim/blob/exim-4.92.1%2Bfixes/doc/doc-txt/ChangeLog
 
diff --git a/debian/changelog b/debian/changelog
index 91a6dc3..d65f363 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,10 +1,10 @@
-exim4 (4.92.2-99a) jessie; urgency=medium
+exim4 (4.92.2-99a) stretch; urgency=medium
 
   * https://github.com/Exim/exim/blob/exim-4.92.2%2Bfixes/doc/doc-txt/ChangeLog
 
  -- kaut <maksimkaut92@gmail.com>  Mon, 9 Aug 2019 12:00:00 +0100
 
-exim4 (4.92.1-99a) jessie; urgency=medium
+exim4 (4.92.1-99a) stretch; urgency=medium
 
   * https://github.com/Exim/exim/blob/exim-4.92.1%2Bfixes/doc/doc-txt/ChangeLog
 
