From e252eb8c71ea3bddb32bf73bddc8b22cfde2bc3a Mon Sep 17 00:00:00 2001
From: Jeremy Harris <jgh146exb@wizmail.org>
Date: Thu, 27 Nov 2014 16:26:44 +0000
Subject: [PATCH] Fix buffer overrun in spam= acl condition.  Bug 1552

---
 src/spam.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/spam.c b/src/spam.c
index 7eb6fbf..76bf7d6 100644
--- a/src/spam.c
+++ b/src/spam.c
@@ -129,7 +129,8 @@ spam(uschar **listptr)
         (spamd_address_container *)store_get(sizeof(spamd_address_container));
 
       /* grok spamd address and port */
-      if( sscanf(CS address, "%s %u", this_spamd->tcp_addr, &(this_spamd->tcp_port)) != 2 ) {
+      if (sscanf(CS address, "%23s %u", this_spamd->tcp_addr, &(this_spamd->tcp_port)) != 2)
+        {
         log_write(0, LOG_MAIN,
           "spam acl condition: warning - invalid spamd address: '%s'", address);
         continue;
-- 
2.1.3

