--- EDITME.orig	2017-02-13 14:45:47.323845720 +0000
+++ EDITME	2017-02-13 14:46:56.857931225 +0000
@@ -131,14 +131,12 @@
 # deliveries. (Local deliveries run as various non-root users, typically as the
 # owner of a local mailbox.) Specifying these values as root is not supported.

-EXIM_USER=ref:exim
-
 # If you specify EXIM_USER as a name, this is looked up at build time, and the
 # uid number is built into the binary. However, you can specify that this
 # lookup is deferred until runtime. In this case, it is the name that is built
 # into the binary. You can do this by a setting of the form:

-# EXIM_USER=ref:exim
+EXIM_USER=ref:Debian-exim

 # In other words, put "ref:" in front of the user name. If you set EXIM_USER
 # like this, any value specified for EXIM_GROUP is also passed "by reference".
@@ -152,8 +150,7 @@
 # for EXIM_USER (e.g. EXIM_USER=exim), you don't need to set EXIM_GROUP unless
 # you want to use a group other than the default group for the given user.

-# EXIM_GROUP=
-EXIM_GROUP=ref:exim
+# EXIM_GROUP=ref:Debian-exim

 # Many sites define a user called "exim", with an appropriate default group,
 # and use
@@ -212,7 +209,7 @@

 # This one is very special-purpose, so is not included by default.

-# ROUTER_IPLOOKUP=yes
+ROUTER_IPLOOKUP=yes


 #------------------------------------------------------------------------------
@@ -244,7 +241,7 @@

 SUPPORT_MAILDIR=yes
 SUPPORT_MAILSTORE=yes
-# SUPPORT_MBX=yes
+SUPPORT_MBX=yes


 #------------------------------------------------------------------------------
@@ -306,14 +303,14 @@
 LOOKUP_DSEARCH=yes
 # LOOKUP_IBASE=yes
 # LOOKUP_LDAP=yes
-# LOOKUP_MYSQL=yes
-LOOKUP_NIS=yes
+LOOKUP_MYSQL=yes
+# LOOKUP_NIS=yes
 # LOOKUP_NISPLUS=yes
 # LOOKUP_ORACLE=yes
 LOOKUP_PASSWD=yes
 # LOOKUP_PGSQL=yes
-# LOOKUP_REDIS=yes
-# LOOKUP_SQLITE=yes
+# LOOKUP_REDIS=yes
+LOOKUP_SQLITE=yes
 # LOOKUP_SQLITE_PC=sqlite3
 # LOOKUP_WHOSON=yes

@@ -373,6 +370,9 @@
 # LOOKUP_LIBS=-L/usr/local/lib -lldap -llber -lmysqlclient -lpq -lgds -lsqlite3


+LOOKUP_INCLUDE=-I/usr/include/mysql
+LOOKUP_LIBS=-lmysqlclient -lsqlite3
+
 #------------------------------------------------------------------------------
 # Compiling the Exim monitor: If you want to compile the Exim monitor, a
 # program that requires an X11 display, then EXIM_MONITOR should be set to the
@@ -381,7 +381,7 @@
 # files are defaulted in the OS/Makefile-Default file, but can be overridden in
 # local OS-specific make files.

-EXIM_MONITOR=eximon.bin
+# EXIM_MONITOR=eximon.bin


 #------------------------------------------------------------------------------
@@ -391,7 +391,7 @@
 # and the MIME ACL. Please read the documentation to learn more about these
 # features.

-# WITH_CONTENT_SCAN=yes
+WITH_CONTENT_SCAN=yes

 #------------------------------------------------------------------------------
 # If you're using ClamAV and are backporting fixes to an old version, instead
@@ -453,9 +453,9 @@
 # installed on your system (www.libspf2.org). Depending on where it is installed
 # you may have to edit the CFLAGS and LDFLAGS lines.

-# EXPERIMENTAL_SPF=yes
+EXPERIMENTAL_SPF=yes
 # CFLAGS  += -I/usr/local/include
-# LDFLAGS += -lspf2
+LDFLAGS += -lspf2

 # Uncomment the following lines to add SRS (Sender rewriting scheme) support.
 # You need to have libsrs_alt installed on your system (srs.mirtol.com).
@@ -468,9 +468,9 @@

 # Uncomment the following line to add DMARC checking capability, implemented
 # using libopendmarc libraries.  You must have SPF support enabled also.
-# EXPERIMENTAL_DMARC=yes
+# EXPERIMENTAL_DMARC=yes
 # CFLAGS += -I/usr/local/include
-# LDFLAGS += -lopendmarc
+# LDFLAGS += -lopendmarc

 # Uncomment the following lines to add Brightmail AntiSpam support. You need
 # to have the Brightmail client SDK installed. Please check the experimental
@@ -497,7 +497,11 @@
 # LDFLAGS += -llmdb

 # Uncomment the following line to add queuefile transport support
-# EXPERIMENTAL_QUEUEFILE=yes
+# EXPERIMENTAL_QUEUEFILE=yes
+
+# Uncomment the following line to add Redis support
+# CFLAGS += -I/usr/local/include
+# LDFLAGS += -lhiredis

 ###############################################################################
 #                 THESE ARE THINGS YOU MIGHT WANT TO SPECIFY                  #
@@ -627,16 +631,16 @@
 # configuration to make use of the mechanism(s) selected.

 AUTH_CRAM_MD5=yes
-# AUTH_CYRUS_SASL=yes
-# AUTH_DOVECOT=yes
+AUTH_CYRUS_SASL=yes
+AUTH_DOVECOT=yes
 # AUTH_GSASL=yes
 # AUTH_GSASL_PC=libgsasl
 # AUTH_HEIMDAL_GSSAPI=yes
 # AUTH_HEIMDAL_GSSAPI_PC=heimdal-gssapi
 # AUTH_HEIMDAL_GSSAPI_PC=heimdal-gssapi heimdal-krb5
 AUTH_PLAINTEXT=yes
-# AUTH_SPA=yes
-# AUTH_TLS=yes
+AUTH_SPA=yes
+AUTH_TLS=yes

 # Heimdal through 1.5 required pkg-config 'heimdal-gssapi'; Heimdal 7.1
 # requires multiple pkg-config files to work with Exim, so the second example
@@ -649,7 +653,7 @@
 # Similarly for GNU SASL, unless pkg-config is used via AUTH_GSASL_PC.
 # Ditto for AUTH_HEIMDAL_GSSAPI(_PC).

-# AUTH_LIBS=-lsasl2
+AUTH_LIBS=-lsasl2
 # AUTH_LIBS=-lgsasl
 # AUTH_LIBS=-lgssapi -lheimntlm -lkrb5 -lhx509 -lcom_err -lhcrypto -lasn1 -lwind -lroken -lcrypt

@@ -689,6 +693,7 @@
 #
 # HAVE_ICONV=yes
 # CFLAGS=-O -I/usr/local/include
+#
 # EXTRALIBS_EXIM=-L/usr/local/lib -liconv
 #
 # but of course there may need to be other things in CFLAGS and EXTRALIBS_EXIM
@@ -908,7 +913,7 @@
 # (version 5.004 or later) installed, set EXIM_PERL to perl.o. Using embedded
 # Perl costs quite a lot of resources. Only do this if you really need it.

-# EXIM_PERL=perl.o
+# EXIM_PERL=perl.o


 #------------------------------------------------------------------------------
@@ -918,7 +923,7 @@
 # that the local_scan API is made available by the linker. You may also need
 # to add -ldl to EXTRALIBS so that dlopen() is available to Exim.

-# EXPAND_DLFUNC=yes
+EXPAND_DLFUNC=yes


 #------------------------------------------------------------------------------
@@ -928,11 +933,11 @@
 # support, which is intended for use in conjunction with the SMTP AUTH
 # facilities, is included only when requested by the following setting:

-# SUPPORT_PAM=yes
+SUPPORT_PAM=yes

 # You probably need to add -lpam to EXTRALIBS, and in some releases of
 # GNU/Linux -ldl is also needed.
-EXTRALIBS=-ldl
+EXTRALIBS=-ldl -lpam -export-dynamic


 #------------------------------------------------------------------------------
@@ -961,7 +964,7 @@
 # If you may want to use inbound (server-side) proxying, using Proxy Protocol,
 # uncomment the line below.

-# SUPPORT_PROXY=yes
+SUPPORT_PROXY=yes


 #------------------------------------------------------------------------------
@@ -1284,7 +1289,7 @@
 # local part) can be increased by changing this value. It should be set to
 # a multiple of 16.

-# MAX_NAMED_LIST=16
+MAX_NAMED_LIST=64


 #------------------------------------------------------------------------------
