--- EDITME.orig	2017-12-01 14:19:09.959704466 +0000
+++ EDITME	2017-12-01 14:22:27.050753809 +0000
@@ -212,7 +212,7 @@
 
 # This one is very special-purpose, so is not included by default.
 
-# ROUTER_IPLOOKUP=yes
+ROUTER_IPLOOKUP=yes
 
 
 #------------------------------------------------------------------------------
@@ -244,7 +244,7 @@
 
 SUPPORT_MAILDIR=yes
 SUPPORT_MAILSTORE=yes
-# SUPPORT_MBX=yes
+SUPPORT_MBX=yes
 
 
 #------------------------------------------------------------------------------
@@ -306,16 +306,16 @@
 LOOKUP_DSEARCH=yes
 # LOOKUP_IBASE=yes
 # LOOKUP_LDAP=yes
-# LOOKUP_MYSQL=yes
-# LOOKUP_MYSQL_PC=mariadb
-LOOKUP_NIS=yes
+LOOKUP_MYSQL=yes
+# LOOKUP_MYSQL_PC=mariadb
+# LOOKUP_NIS=yes
 # LOOKUP_NISPLUS=yes
 # LOOKUP_ORACLE=yes
 LOOKUP_PASSWD=yes
 # LOOKUP_PGSQL=yes
 # LOOKUP_REDIS=yes
-# LOOKUP_SQLITE=yes
-# LOOKUP_SQLITE_PC=sqlite3
+LOOKUP_SQLITE=yes
+# LOOKUP_SQLITE_PC=sqlite3
 # LOOKUP_WHOSON=yes
 
 # These two settings are obsolete; all three lookups are compiled when
@@ -374,6 +374,9 @@
 # LOOKUP_LIBS=-L/usr/local/lib -lldap -llber -lmysqlclient -lpq -lgds -lsqlite3
 
 
+LOOKUP_INCLUDE=-I/usr/include/mysql
+LOOKUP_LIBS=-lmysqlclient -lsqlite3
+
 #------------------------------------------------------------------------------
 # Compiling the Exim monitor: If you want to compile the Exim monitor, a
 # program that requires an X11 display, then EXIM_MONITOR should be set to the
@@ -382,7 +385,7 @@
 # files are defaulted in the OS/Makefile-Default file, but can be overridden in
 # local OS-specific make files.
 
-EXIM_MONITOR=eximon.bin
+# EXIM_MONITOR=eximon.bin
 
 
 #------------------------------------------------------------------------------
@@ -392,7 +395,7 @@
 # and the MIME ACL. Please read the documentation to learn more about these
 # features.
 
-# WITH_CONTENT_SCAN=yes
+WITH_CONTENT_SCAN=yes
 
 #------------------------------------------------------------------------------
 # If you're using ClamAV and are backporting fixes to an old version, instead
@@ -454,9 +457,9 @@
 # installed on your system (www.libspf2.org). Depending on where it is installed
 # you may have to edit the CFLAGS and LDFLAGS lines.
 
-# EXPERIMENTAL_SPF=yes
+EXPERIMENTAL_SPF=yes
 # CFLAGS  += -I/usr/local/include
-# LDFLAGS += -lspf2
+LDFLAGS += -lspf2
 
 # Uncomment the following lines to add SRS (Sender rewriting scheme) support.
 # You need to have libsrs_alt installed on your system (srs.mirtol.com).
@@ -500,6 +503,10 @@
 # Uncomment the following line to add queuefile transport support
 # EXPERIMENTAL_QUEUEFILE=yes
 
+# Uncomment the following line to add Redis support
+# CFLAGS += -I/usr/local/include
+# LDFLAGS += -lhiredis
+
 ###############################################################################
 #                 THESE ARE THINGS YOU MIGHT WANT TO SPECIFY                  #
 ###############################################################################
@@ -628,16 +635,16 @@
 # configuration to make use of the mechanism(s) selected.
 
 AUTH_CRAM_MD5=yes
-# AUTH_CYRUS_SASL=yes
-# AUTH_DOVECOT=yes
+AUTH_CYRUS_SASL=yes
+AUTH_DOVECOT=yes
 # AUTH_GSASL=yes
 # AUTH_GSASL_PC=libgsasl
 # AUTH_HEIMDAL_GSSAPI=yes
 # AUTH_HEIMDAL_GSSAPI_PC=heimdal-gssapi
 # AUTH_HEIMDAL_GSSAPI_PC=heimdal-gssapi heimdal-krb5
 AUTH_PLAINTEXT=yes
-# AUTH_SPA=yes
-# AUTH_TLS=yes
+AUTH_SPA=yes
+AUTH_TLS=yes
 
 # Heimdal through 1.5 required pkg-config 'heimdal-gssapi'; Heimdal 7.1
 # requires multiple pkg-config files to work with Exim, so the second example
@@ -650,7 +657,7 @@
 # Similarly for GNU SASL, unless pkg-config is used via AUTH_GSASL_PC.
 # Ditto for AUTH_HEIMDAL_GSSAPI(_PC).
 
-# AUTH_LIBS=-lsasl2
+AUTH_LIBS=-lsasl2
 # AUTH_LIBS=-lgsasl
 # AUTH_LIBS=-lgssapi -lheimntlm -lkrb5 -lhx509 -lcom_err -lhcrypto -lasn1 -lwind -lroken -lcrypt
 
@@ -690,6 +697,7 @@
 #
 # HAVE_ICONV=yes
 # CFLAGS=-O -I/usr/local/include
+#
 # EXTRALIBS_EXIM=-L/usr/local/lib -liconv
 #
 # but of course there may need to be other things in CFLAGS and EXTRALIBS_EXIM
@@ -919,7 +927,7 @@
 # that the local_scan API is made available by the linker. You may also need
 # to add -ldl to EXTRALIBS so that dlopen() is available to Exim.
 
-# EXPAND_DLFUNC=yes
+EXPAND_DLFUNC=yes
 
 
 #------------------------------------------------------------------------------
@@ -929,11 +937,11 @@
 # support, which is intended for use in conjunction with the SMTP AUTH
 # facilities, is included only when requested by the following setting:
 
-# SUPPORT_PAM=yes
+SUPPORT_PAM=yes
 
 # You probably need to add -lpam to EXTRALIBS, and in some releases of
 # GNU/Linux -ldl is also needed.
-EXTRALIBS=-ldl
+EXTRALIBS=-ldl -lpam -export-dynamic
 
 
 #------------------------------------------------------------------------------
@@ -947,7 +955,7 @@
 # If you may want to use inbound (server-side) proxying, using Proxy Protocol,
 # uncomment the line below.
 
-# SUPPORT_PROXY=yes
+SUPPORT_PROXY=yes
 
 
 #------------------------------------------------------------------------------
@@ -1285,7 +1293,7 @@
 # local part) can be increased by changing this value. It should be set to
 # a multiple of 16.
 
-# MAX_NAMED_LIST=16
+MAX_NAMED_LIST=64
 
 
 #------------------------------------------------------------------------------
