--- EDITME.ORIG 2020-05-30 20:35:38.000000000 +0000
+++ EDITME      2020-08-12 14:56:33.739439295 +0000
@@ -1,6 +1,7 @@
 ##################################################
 #                The Exim Monitor                #
 ##################################################
+# -*- makefile -*-
 
 # This is the template for the Exim monitor's main build-time configuration
 # file. It contains settings that are independent of any operating system. It
